src/test/java/com/hocs/test/pages/ukvi/MPAMMultipleContributions.java

```diff
@@ -51,8 +52,14 @@ public void sendMPAMCaseToContributionRequest() {
     }
 
     public void addMultipleContributionRequests(int numberOfContributions) {
```

Comment:
We need to refactor this so that it makes sense for both MPAM and FOI. We should either create a separate file for FOI, or move this method to a file thats in the CS folder.

Reply:
See comment regarding Approval requests